paddle/fluid/framework/new_executor/program_interpreter.cc

```diff
@@ -283,6 +283,17 @@ void ProgramInterpreter::ShareWorkQueueFrom(InterpreterBaseImpl* src) {
           << ") to InterpreterCore(" << this << ")";
 }
 
+void ProgramInterpreter::ShareBuildResultsFrom(InterpreterBaseImpl* src) {
```

Comment:
```suggestion
void ProgramInterpreter::ShareBuildResultsFrom(const InterpreterBaseImpl& src) {
```
非可选输入参数通常是值参或 const 引用, 非可选输出参数或输入/输出参数通常应该是引用 （不能为空）.

Reply:
done